_pytest/recwarn.py

```diff
@@ -115,19 +114,14 @@ def warns(expected_warning, *args, **kwargs):
             return func(*args[1:], **kwargs)
 
 
-RecordedWarning = namedtuple('RecordedWarning', (
```

Comment:
No, that was added by you in features branch for 3.1, so it has never seen the light of day.